pack.yaml

```
@@ -1,11 +1,12 @@
 ref: cisco_aci
 name: cisco_aci
 description: Cisco ACI pack
-version: 0.1.6
+version: 0.1.7
```

```suggestion
version: 0.2.0
```

Considering this PR is not a bugfix, but a feature with new changes, please bump the version accordingly to https://semver.org/#summary